### ISSUE TITLE
Could not find the following native modules: fullstory_react-native

### ISSUE THREAD

**Issue by kamal2677** [fullstorydev/fullstory-react-native#4]
Unable to use plugin 
getting following error while installing pod.

![Screenshot 2020-05-07 at 2 03 27 PM](https://user-images.githubusercontent.com/8983733/81272756-8970e880-906b-11ea-853a-31bbf76200cc.png)

Also the example code provided is not working.

**Comment by kamal2677** [comment]
Hello there, Is their any update on above query ?

**Comment by devladinci** [comment]
@kamal2677 Add FullStory pod to your Podfile `  pod 'FullStory', :http => 'https://ios-releases.fullstory.com/fullstory-1.2.0.tar.gz'`

**Comment by JoshMiers-FS** [comment]
Hey there!

Thanks for reaching out and filing an issue with us here on Github. Our react native plugin is currently in private beta which is why you may find the documentation a bit hard to follow. If you’re interested in gaining access to the private beta, please email mobile-support@fullstory.com and we’ll follow up with next steps.

While we wish we could grant everyone access to the beta program, please bear in mind that we’re evaluating each request on a case-by-case basis and admission into the beta is not guaranteed. Once we’re out of private beta and access to our react-native plugin is more widely available, we will update our documentation with steps on how to integrate it properly into your project.

Sorry for any confusion this has caused and thank you for your patience while we work to make this plugin great for the entire community.

**Comment by raimonkh** [comment]
Hello, 
I'm having trouble as well with the react-native installation.
After removing `use_frameworks` from podfile (that was causing a transitive dependency) I get the following on build `ios/FullStory.h:2:9: 'FullStory/FS.h' file not found`

**Comment by mattjbrill** [comment]
> Hello,
> I'm having trouble as well with the react-native installation.
> After removing `use_frameworks` from podfile (that was causing a transitive dependency) I get the following on build `ios/FullStory.h:2:9: 'FullStory/FS.h' file not found`

I ran into the same issue. What resolved it for me was to clean my project, made sure my pod file is set properly (https://help.fullstory.com/hc/en-us/articles/360042772333-Getting-Started-with-iOS-Recording), then pod install.

**Comment by sabrina-li** [comment]
Thanks for the posts! This plug-in is intended to be used in conjunction with FullStory for Mobile Apps. To add FullStory for Mobile Apps for Android and iOS, checkout our getting started guide: [Android](https://help.fullstory.com/hc/en-us/articles/360040596093-Getting-Started-with-Android-Recording), [iOS](https://help.fullstory.com/hc/en-us/articles/360042772333-Getting-Started-with-iOS-Recording) and [React Native](https://help.fullstory.com/hc/en-us/articles/360052419133-Getting-Started-with-FullStory-React-Native-Recording)

If you experience any issues in the future, feel free to reach out to mobile-support@fullstory.com if you need any assistance with the React Native plugin or other Mobile recording support.
